src/librustc_metadata/diagnostics.rs

```diff
@@ -91,6 +91,204 @@ You need to link your code to the relevant crate in order to be able to use it
 well, and you link to them the same way.
 "##,
 
+E0466: r##"
+Invalid macro import declarations.
+
+Causes of this error:
```

Comment:
Is the language that critical? 

Just looking at handful random error numbers, it looks like E0050, E0053, E0054, and E0055 don't say "Erroneous code example".

If we stick that strictly to the RFC it looks like some old errors need to be rewritten.